src/views/bootablevolumes/utils/utils.ts

```diff
@@ -105,9 +105,8 @@ export const getInstanceTypesToSizesMap = (
   instanceTypesNames.reduce((instanceTypesAndSizes, instanceType) => {
     const [instanceTypePart, sizePart] = instanceType.split('.');
 
-    instanceTypesAndSizes[instanceTypePart] !== undefined
```

Comment:
The logic is the same here?
In the old code, if it exists, you are replacing the array, and in the new code, you simply push if it exists. still, working as it should?